### DISCUSSION TITLE
Symbol(pinia)" not found

### DISCUSSION THREAD

**Discussion by rangeldor** [vuejs/pinia#713] [Help and Questions] [answered]
![image](https://user-images.githubusercontent.com/51749346/136637615-0c88609b-a4e7-4a83-86ed-8f036455278e.png)

I got this error and I don't know why. 

My SignIn.vue

![image](https://user-images.githubusercontent.com/51749346/136637745-0240e72b-174e-459c-afc9-e16d6d8fd572.png)

I am using pinia 2.0.0-rc.12

**Comment by posva** [comment, marked as answer]
You seem to be missing the app.use(pinia) in your main.ts like shown in docs.

**Reply by rangeldor** [reply]
Thanks @posva !

 I think pinia have been removed when I add vuetify in my project.

**Reply by rangeldor** [reply]
Curious fact... It still worked in my project, it just didn't show up in devtool.

**Reply by posva** [reply]
Maybe you were calling `useSomeStore()` before `app.use(pinia)` (somehow)
